### ISSUE TITLE
Minor suggestions

### ISSUE THREAD

**Issue by hunan-rostomyan** [dvx/fonz.js#1]
Thanks for the useful tool. Minor suggestions follow.

1. `phone` is [assumed](https://github.com/dvx/fonz.js/blob/master/index.js#L51) to be a string so [this](https://github.com/dvx/fonz.js/blob/master/index.js#L59) isn't necessary. It is defensive though, so maybe not a bad idea. Thought I'd bring to your attention in case it's accidental.
2. There is some inconsistency in the comparisons: you rely on coercion [here](https://github.com/dvx/fonz.js/blob/master/index.js#L64) when ensuring the country code is 1, but in the rest of the code you explicitly compare string slices with strings. It might save some trouble in the future if you switch from `==`s to `===`s in general and here do `phone[0] !== '1'`. But you know what you're doing, so I'm only half-suggesting the strict equality move.
3. The following are equivalent for all k, so you can simplify [this](https://github.com/dvx/fonz.js/blob/master/index.js#L67).
    ```
    str.substring(k, str.length)
    str.substring(k)
    ````
    Admittedly less explicit, so questionable whether this is an improvement.
4. Could simplify [this](https://github.com/dvx/fonz.js/blob/master/index.js#L85) to:
    ```js
    exchange_code.substring(1) !== '11'
    ```

**Comment by dvx** [comment]
Hey Hunan, thanks for the feedback!

- (1) is indeed defensive. If `phone` is not a string, `.replace()` will throw an exception.
- (2) is a mistake. Everything should be using `==` with strings on both sides, so I'll fix this. I don't particularly like using `===` with strings as it has (in some cases) weird behavior, e.g.: `"abc" == new String("abc")` will evaluate to `true` but `"abc" === new String("abc")` will evaluate to `false`.
- (3) I had no idea about 🙊
- (4) is a good change which I'll patch in.

Thanks again!

**Comment by dvx** [comment]
Above changes merged as part of version [1.0.2](https://github.com/dvx/fonz.js/releases/tag/v1.0.2). Closing.
